### PR TITLE
jd-gui: new port

### PR DESCRIPTION
**Pull request by lhaeger**
###### Tested on

macOS 10.14.6 18G1012
Xcode 11.2 11B52

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: java/jd-gui/Portfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+homepage                http://java-decompiler.github.io/
+
+checksums               rmd160  4224379fa8a97854ea8e94b5d71e4fe715ea2c32 \
+                        sha256  f10ff52e6abe19381d453391aa6cfebdda4b579a1e926aca7dc310b6d0bcda2b \
+                        size    365193
+                        
+java.version            1.8+
+java.fallback           openjdk11
+
+use_configure           no
```
Comment (mf2k) [inline java/jd-gui/Portfile]: The universal variant is not likely to work and should probably be removed. 

`universal_variant no
`

Comment (lhaeger) [reply java/jd-gui/Portfile]: +1

**Comment by mf2k** [comment]
The commit message should be more simply:

`jd-gui: new port
`

**Comment by mf2k** [comment]
Buildbot error is:

`Failed to parse file java/jd-gui/Portfile: invalid command name "q#"
`

**Pull request merged**